requirements.in

```diff
@@ -1,9 +1,11 @@
--e .
```

Comment:
Why are you abandoning pulling in the `setup.py` requirements?
Doesn't this duplicate information between here and `setup.py`?

Reply:
Yes, you're right this duplicates information. Will fix this.